docs/develop/mainnet/api.md

```diff
@@ -0,0 +1,3 @@
+# Mainnet API
```

Comment:
@masterfung I removed this page on a previous PR and moved the content to https://main--dancing-hummingbird-242a98.netlify.app/develop/mainnet. 

So I'd keep docs/develop/mainnet/api.md removed

Reply:
Sounds good